app-tasks/rf/src/rf/uploads/geotiff/create_footprints.py

```diff
@@ -143,8 +143,11 @@ def extract_polygon(mask_tif_path):
     mask = np.ma.masked_equal(raster, 0)
     logger.info('Extracting shapes from footprint masks')
     geoms = shapes(raster, mask=mask.astype('bool'), transform=src_affine, connectivity=4)
+    try:
+        footprint, value = geoms.next()
+    except StopIteration:
+        raise Exception('Shapes could not be extracted from the footprint mask, the file could be invalid')
```

Comment:
This could probably be a more specific error. a `ValueError` maybe? Or an `IOError`? Or maybe just `Exception` is fine since none of the built-ins appear to cover this specific case